.scalafmt.conf

```diff
@@ -1,5 +1,6 @@
-version=2.4.2
+version="3.0.5"
```

Comment:
So I've attempted to resolve the plugin error by removing the offending `edition`. This ended up causing a whole slew of syntactic changes (expected), though the main thing I object to is

```diff
@@ -167,7 +160,7 @@ object SwaggerUtil {

       for {
         customTpe <- customType.indexedDistribute.flatTraverse(x => liftCustomType[L, F](x))
-        result <- customTpe.fold({
+        result <- customTpe.fold {
           (typeName.unwrapTracker, format.unwrapTracker) match {
             case (Some("string"), Some("uuid"))         => uuidType()
             case (Some("string"), Some("password"))     => stringType(None)
@@ -193,7 +186,7 @@ object SwaggerUtil {
             case (tpe, fmt) =>
               fallbackType(tpe, fmt)
           }
-        })(_.pure[F])
+        }(_.pure[F])
         _ <- Sw.log.debug(s"Returning ${result}")
       } yield result
     }
```
The syntax around `x.fold { foo }(_.bar)` is very unpleasant to me, I'd prefer how it was previously, `x.fold({ foo })(_.bar)`.

Do you know how to do this? If not, can the scalafmt change be deferred? I don't have a problem with reformatting the whole repo, managing complex PRs through major reformats is why I wrote and maintain [`git-rebase-formatting`](https://github.com/blast-hardcheese/git-rebase-formatting), but I would prefer to trend towards more usable and tractable style instead of a regression like this.

Reply:
That's a nice script.

The formatting sounds like it should be part only of the redundant parenthesis re-write, I'd have to double-check. If not, I'll remove the config change.

Reply:
I removed the scalafmt file -- the diff of scalafmtAll was too unwieldly to smuggle in under the radar.